app/main/helpers/validation.py

```diff
@@ -96,6 +98,16 @@ def character_limit_errors(self):
 
         return errors_map
 
+    def word_limit_errors(self):
+        errors_map = {}
+        for question_id in self.all_fields():
+            if self.content.get_question(question_id).get('type') in ['text', 'textbox_large']:
+                answer = self.answers.get(question_id) or ''
```

Comment:
```suggestion
                answer = self.answers.get(question_id, '')
```

Reply:
Good ol' copy and paste there! Will amend

Reply:
Ah - we do need this, as `question_id` may be present in `self.answers` but still return `None`, and we want to validate that.

Reply:
Right, got it, my bad